controllers/profiles/removeImageProfile.js

```diff
@@ -20,13 +20,16 @@ module.exports = async (req, res) => {
     } else {
       let myTs = moment(new Date()).format('YYYY-MM-DD HH:mm:ss');
 
-      updateUser(req?.userId, {
+      await updateUser(req?.userId, {
         username: user.username,
-        profile_pic_url: null
+        profile_pic_url: defaultImage,
+        profileImage: defaultImage,
+        image: defaultImage
       });
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

It seems like you're setting the same `defaultImage` to three different properties (`profile_pic_url`, `profileImage`, and `image`). If these properties are meant to represent the same thing, consider consolidating them into a single property to avoid redundancy and potential inconsistencies.



<!-- This is an auto-generated comment by OSS CodeRabbit -->